app/src/sharedTest/java/org/oppia/android/app/topic/TopicFragmentTest.kt

```diff
@@ -652,10 +807,41 @@ class TopicFragmentTest {
     )
   }
 
+  private fun markAllSpotlightsSeen() {
+    markLessonsTabSpotlightSeen()
+    markFirstChapterSpotlightSeen()
+    markRevisionTabSpotlightSeen()
+  }
+
+  private fun markRevisionTabSpotlightSeen() {
+    val profileId = ProfileId.newBuilder()
+      .setInternalId(internalProfileId)
+      .build()
+    spotlightStateController.markSpotlightViewed(profileId, TOPIC_REVISION_TAB)
+    testCoroutineDispatchers.runCurrent()
+  }
+
+  private fun markLessonsTabSpotlightSeen() {
+    val profileId = ProfileId.newBuilder()
+      .setInternalId(internalProfileId)
+      .build()
+    spotlightStateController.markSpotlightViewed(profileId, TOPIC_LESSON_TAB)
+    testCoroutineDispatchers.runCurrent()
+  }
+
+  private fun markFirstChapterSpotlightSeen() {
+    val profileId = ProfileId.newBuilder()
+      .setInternalId(internalProfileId)
+      .build()
+    spotlightStateController.markSpotlightViewed(profileId, FIRST_CHAPTER)
+    testCoroutineDispatchers.runCurrent()
+  }
```

Comment:
These can be simplified by using a common helper--please update.

Reply:
done